SportsGoods.Data/DAL/ApplicationDbContext.cs

```diff
@@ -2,45 +2,32 @@
 using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore;
 using SportsGoods.Core.Interfaces;
-using SportsGoods.Data.Models;
+using SportsGoods.Core.Models;
+using SportsGoods.Data.DAL.EntityConfiguration;
+using System.Reflection;
 
 namespace SportsGoods.Data.DAL
 {
-    public class ApplicationDbContext : IdentityDbContext<Administrator>, IDbContext
+    public class ApplicationDbContext : IdentityDbContext<Administrator>, IApplicationDbContext
     {
         public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
             : base(options)
-        {
-        }
+        { }
+        public ApplicationDbContext() : base()
+        { }
+
         public DbSet<Administrator> Administrators { get; set; }
+        public DbSet<Product> Products { get; set; }
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
+            modelBuilder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());
             base.OnModelCreating(modelBuilder);
-
-            modelBuilder.Entity<Administrator>().ToTable("Administrators");
-
-            modelBuilder.Entity<Administrator>().HasData(
-                new Administrator
-                {
-                    Id = new Guid(),
-                    Email = "FirstAdmin@abv.bg",
-                    Password = "password1"
-                },
-                 new Administrator
-                 {
-                     Id = new Guid(),
-                     Email = "SecondAdmin@abv.bg",
-                     Password = "password2"
-                 },
-                  new Administrator
-                  {
-                      Id = new Guid(),
-                      Email = "ThirdAdmin@abv.bg",
-                      Password = "password3"
-                  }
-            );
         }
 
```

Comment:
if it's commented out then it's not needed

Reply:
Done (removed).